### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gramster**
Add mention of stubs repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -187,6 +187,8 @@ Pylance leverages Microsoft's open-source static type checking tool, Pyright, to
 
 Code contributions are welcomed via the [Pyright](https://github.com/microsoft/pyright) repo.
 
+Pylance also makes use of some type stub files, either from typeshed, or some additional ones that we eventually hope to move either inline in the source packages or to typeshed. You can find and contribute to our work-in-progress type stubs at https://github.com/microsoft/python-type-stubs.
```
Comment (savannahostrowski) [inline README.md:190]: Thoughts?

> Pylance ships with a collection of type stubs for popular modules to provide fast and accurate auto-completions and type checking. Our type stubs are sourced from [typeshed](https://github.com/python/typeshed) and our work-in-progress stub repository, [microsoft/python-type-stubs]( https://github.com/microsoft/python-type-stubs). Type stubs in microsoft/python-type-stubs will be contributed back to typeshed or added inline to source packages once they are of high enough quality.

**Review by savannahostrowski — CHANGES_REQUESTED** [review]

**Comment by alberto301230** [comment]
Para el problema del import en python a mi lo que me generaba el problema era la extensión Pylance. Lo que hice para solucionarlo: En el settings.json buscar y deshabilitar la línea que dice “python.languageServer”: ‘Pylance’" y listo, espero les funcione.

Nota: Para los novatos como yo en VSCODE, para abrir el settings.json, ingresan a la paleta de comandos Ctrl+Shift+P y buscan Open Settings (JSON)

File: README.md (hunk 1 above)
Comment (savannahostrowski) [reply README.md:190]: Pinging this thread @gramster

**Comment by savannahostrowski** [comment]
Merged these changes in via #1275

**Pull request closed without merging**